### PR TITLE
Page titles

### PR DESCRIPTION
**Pull request by martinremis**
## Summary
Add a title "component".

## Screenshot
page title

## Trello Card
https://trello.com/c/UMAUGi7K/21-page-titles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <div class="titles">
+    <img src="assets/triangle.svg" class="titles-img"/>
```
Comment (FrankIglesias) [inline index.html]: add an alt to the image

File: scss/styles.scss [hunk 2]
```diff
@@ -1,3 +1,15 @@
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
 .title {
-  color: red;
+  align-self: center;
+  font-size: 1.5em;
```
Comment (FrankIglesias) [inline scss/styles.scss]: use px dimensions

File: scss/titles.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.titles {
+  display: flex;
+  flex-flow: row nowrap;
```
Comment (FrankIglesias) [inline scss/titles.scss]: `flex-direction: row`

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: scss/styles.scss [hunk 4]
```diff
@@ -1,3 +1,15 @@
+* {
```
Comment (FrankIglesias) [inline scss/styles.scss]: move this into equalizer

File: scss/styles.scss [hunk 5]
```diff
@@ -1,3 +1,15 @@
...
+}
+
 .title {
-  color: red;
+  align-self: center;
+  font-size: 1.5em;
+  margin-left: -33px;
+}
+
+.titles-img {
```
Comment (FrankIglesias) [inline scss/styles.scss]: I think you don't need flex: 0 1;

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by FrankIglesias** [comment]
Also, define the images next to the titles as pseudo elements

**Comment by martinremis** [comment]
I've also changed the content of style.scss to titles.scss and removed an unused css class (titles).

File: index.html [hunk 6]
```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <div class="titles">
```
Comment (FrankIglesias) [inline index.html]: I think that now there's no need of this external div

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**